pkgs/applications/networking/pyload/default.nix

```diff
@@ -0,0 +1,68 @@
+{ stdenv, fetchFromGitHub, fetchpatch, python, gocr, unrar, rhino, spidermonkey }:
+let
+  pythonEnv = (python.withPackages (ps: with ps; [
+    pycurl jinja2 beaker thrift simplejson
+    #optional packages
+    #TODO add Send2Trash
+    #TODO package the other optional ones, see homepage
+    pycrypto feedparser pyqt4
+  ]));
+in
+stdenv.mkDerivation rec {
+  version = "stable";
```

Comment:
According to the Nixpkgs guidelines `stable` is not a version. I suggest you use the latest release.